README.md

```diff
@@ -229,33 +206,25 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [syringe](https://github.com/Bonemind/syringe) - A simple and basic dependency injection shard for crystal
 
 ## Development Tools
```

Comment:
Please remove an empty section and clean TOC

Reply:
I'll do that before merging.  So that the sections are still there if people fix their shard.